org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -83,6 +83,33 @@ class Translator {
     }
 
     /**
+     * Subscribe to notifications for a platform.
+     * This function is intended to be called by the platform translator for initial subscription,
+     * and on the hub translator (this) for verification.
+     */
+    postSubscribe(subscriptionInfo) {
```

Comment:
>subscriptionInfo [](start = 18, length = 16)

what's the shape of this object? only controlId? i don't see this called out in the raml